### PR TITLE
Add ndc-go v1.2.3

### PR DESCRIPTION
**Pull request by hgiasac**
Adds the v1.2.3 and v0.6.3 of the Go connector to the hub registry.

### CODE REVIEW

File: registry/go/metadata.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "title": "Go Connector",
     "logo": "logo.svg",
     "tags": [],
-    "latest_version": "v1.1.0"
+    "latest_version": "v1.2.0"
```
Comment (scriptnull) [inline registry/go/metadata.json:8]: should this be `v1.2.3` instead?

Comment (hgiasac) [reply registry/go/metadata.json:8]: yes, correct. I changed new versions and forgot this

**Review by scriptnull — APPROVED** [review]
Published v1.2.3 :heavy_check_mark:

**Comment by scriptnull** [comment]
I will publish v0.6.3 too now and post an update here.

**Comment by scriptnull** [comment]
Published v0.6.3 too :heavy_check_mark:

**Pull request merged**